streaming/spark-streaming/examples/consumer/manifests/01_spark_application.yaml

```diff
@@ -0,0 +1,87 @@
+---
+apiVersion: "sparkoperator.k8s.io/v1beta2"
+kind: SparkApplication
+metadata:
+  name: spark-consumer
+  namespace: spark-operator
+spec:
+  type: Python
+  pythonVersion: "3"
+  mode: cluster
+  image: "public.ecr.aws/data-on-eks/consumer-spark-streaming-3.3.2-kafka:1"
```

Comment:
Add a note on how users can create/customize this  image here by pointing to the Dockerfile etc.